src/components/pokemonListCard.tsx

```diff
@@ -25,20 +25,20 @@ export default function PokemonListCard({
         <div
             className={`${getBackgroundColor(
                 pokemon.types[0]!.type.name as Type
-            )} items-center justify-between p-1.5 flex border-2 border-black rounded-3xl transition-all duration-500 ${
+            )} items-center justify-between p-1.5 flex border-2 border-black rounded-3xl transition-all duration-500 w-fit ${
```

Comment:
`w-fit` is not necessary here. In general `w-fit` is not necessary in most scenarios. If you feel it is, you did something wrong somewhere else most of the times.

In this case this div is getting limited in its width by its parent element because you used a wrong col-span there.